docs/developer/payments/lifecycle.mdx

```diff
@@ -314,3 +334,39 @@ for `transaction.canceledAmount` will not take into account the amount from `CAN
 If an `CANCEL_FAILURE` event is related to an `CANCEL_REQUEST` event by `pspReference`, then the `transactionItem.cancelPendingAmount`
 will be reduced by the amount reported in the `CANCEL_FAILURE` event. If there is no related `CANCEL_REQUEST`, the failure event will only
 be used to provide the history of `transactionItem` actions.
+
+
+## Transaction event flow
+
+Transaction events modify the amount fields, which define the state of the transaction.
+We do not require any specific order of up comming events, the transaction amounts will be calculated
```

Comment:
"We do not require any specific order of up comming events"
should be:
"We do not require any specific order of incoming events"